packages/crayons-core/src/components/options-list/list-options.tsx

```diff
@@ -265,8 +265,10 @@ export class ListOptions {
 
   @Method()
   async setSelectedOptions(options: any[]): Promise<any> {
```

Comment:
@rihansiddhi do we have any case where someone can set `setSelectedOptions(null)`.
Can you please validate this ?

Reply:
@arvindanta Yes, we are setting selected options as null using setSelectedOptions method for some cases which will be impacted.

CC @abisht1991

Reply:
@arvindanta @rihansiddhi 
This would do right, will make the changes.

`if (options || options === null) {
      this.selectedOptionsState = options;
      this.setValue(options);
    }`

Reply:
@abisht1991 Will checking `if (options !== undefined)` suffice your usecase?

Reply:
yes that would also work @rihansiddhi